Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -94,6 +100,18 @@ IndicatorInfoProvider {
             }
         }
     }
+    func composeIndicatorTitle(with itemCount: Int) -> String {
+        var newTitle: String
```

Comment:
This can be a constant (`let`), as it is inferred to be initialized only once. :)

Reply:
👍